AuxTel/Troubleshooting/ATCS/AuxTel-Dome-Fails-to-Arrive-in-Position.rst

```diff
@@ -0,0 +1,187 @@
+.. This is a template for troubleshooting when some part of the observatory enters an abnormal state. This comment may be deleted when the template is copied to the destination.
+
+.. Review the README in this procedure's directory on instructions to contribute.
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README in this procedure's directory on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Erik Dennihy*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Ioana Sotuela, Gonzalo Aravenas*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _AuxTel-Dome-Fails-to-Arrive-in-Position:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+#########################
+AuxTel Dome Fails to Arrive in Position
+#########################
+
+
+.. _AuxTel-Dome-Fails-to-Arrive-in-Position-Overview:
+
+Overview
+========
+
+.. In one or two sentences, explain when this troubleshooting procedure needs to be used. Describe the symptoms that the user sees to use this procedure. 
+
+When slewing to a new target the Dome stops tracking and fails to arrive in its final position, resulting in script failure (if the slew was triggered from a script) and the ATPtg CSC going into ``FAULT``. 
+The ATPtg can be recovered by performing a state cycle on LOVE, but the dome following must also be cycled independently to recover dome tracking. 
```

Comment:
- Change lower caps in dome for Dome.
   * but the dome following must also be cycled.
* Dome Following ? can also be comment that refers to  ATDomeTrajectory CSC element